setup.cfg

```diff
@@ -1,4 +1,7 @@
 [flake8]
+ignore =
+    E203
+    W503
```

Comment:
[question] Why do you add those ignores?

Reply:
This is because of the conflict between black formatter and flake8.

E203 enforces us to code like this:

```python
n = 1
a = [1, 2, 3]
a[:n+1]
```

However, black requires the code to look like this:

```python
n = 1
a = [1, 2, 3]
a[: n + 1]
```

By ignoring E203, we can avoid the confliction, but we also need to ignore W503 to completely ignore the error.
This is the reason why we need them.

We have the same setup in Optuna as well. ([ref](https://github.com/optuna/optuna/blob/master/setup.cfg#L4-L5)).